pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -110,52 +105,65 @@ func (c *Cluster) configureDashboardModules() error {
 	return nil
 }
 
-func (c *Cluster) configureDashboardModuleSettings(daemonID string) (bool, error) {
+func (c *Cluster) configureDashboardModuleSettings() (bool, error) {
+
 	monStore := config.GetMonStore(c.context, c.clusterInfo)
 
-	daemonID = fmt.Sprintf("mgr.%s", daemonID)
+	// Remove any existing per mgr-daemon configuration
+	if removeMgrDaemonConfiguration {
+		for _, daemonID := range c.getDaemonIDs() {
```

Comment:
How about an overall simple log statement to indicate that the old settings are being removed if they exist. It's helpful for troubleshooting in the operator log what is happening.

Reply:
Done :+1: